.github/workflows/build-wasm.yml

```diff
@@ -56,3 +57,20 @@ jobs:
           name: sdk-bitwarden-wasm
           path: ${{ github.workspace }}/languages/js/wasm/*
           if-no-files-found: error
+
+      - name: Set version
+        if: ${{ github.ref == 'refs/head/main' }}
+        # Fetches current version from registry and uses prerelease to bump it
+        run: |
+          npm version --no-git-tag-version $(npm view @bitwarden/sdk-wasm@latest version)
+          npm version --no-git-tag-version prerelease
+        env:
+          NODE_AUTH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        working-directory: languages/js/wasm
+
+      - name: Publish NPM
+        if: ${{ github.ref == 'refs/head/main' }}
+        run: npm publish
```

Comment:
I think this will need to be `npm publish --access public`, otherwise the dependency will be private and only accessible to members of the bitwarden organization, as it's scoped under the @bitwarden name